checker/src/org/checkerframework/checker/oigj/ImmutabilityAnnotatedTypeFactory.java

```diff
@@ -4,6 +4,7 @@
 import org.checkerframework.checker.oigj.qual.I;
 import org.checkerframework.checker.oigj.qual.Immutable;
 import org.checkerframework.checker.oigj.qual.Mutable;
+import org.checkerframework.checker.oigj.qual.OIGJMutabilityBottom;
 import org.checkerframework.checker.oigj.qual.ReadOnly;
```

Comment:
This one is actually need because I moved OIGJMutabilityBottom to the qual package.